### ISSUE TITLE
clozestrike

### ISSUE THREAD

**Issue by atmicmic** [Josef-Friedrich/cloze#8]
You need the Transparent Package just for this line in clozestrike??
 {\texttransparent{0}{\clozefont{}#3}}
You could consider instead something like
{\phantom{\clozefont{}#3}}
i have problems with transparent package.

Also there is a problem with \ifclozeshow in \clozestrike, but i dont know where its coming from , some other commands of your package seem to  set it to TRUE and i need an extra \clozehide just before using clozestrike.

Also ulem Package seems just to load for the clozestrikeline. That is strange because your whole package is about a Lua-way of  of underlining ... there is also a  package LuauL which maybe could be "integrated" or "combined"

The clozestrike command is kind of "nice" but it absolutly doesn't fit to rest of this package but does need extra dependencies. 
Best whishes!

**Comment by Josef-Friedrich** [comment]
You are absolutely right. This small command `\clozestrike` should not require any additional dependencies. Hopefully I find time to implement it in Lua.

**Comment by Josef-Friedrich** [comment]
I have just released version [v1.8.1](https://github.com/Josef-Friedrich/cloze/releases/tag/v1.8.1). The `\clozestrike` command is now implemented in pure Lua. The dependencies to the packages [transparent](https://www.ctan.org/pkg/transparent), [stackengine](https://www.ctan.org/pkg/stackengine) and [ulem](https://www.ctan.org/pkg/ulem) are removed.
